src/app/domains/wallets/domains.wallets.gateway.ts

```diff
@@ -26,4 +27,20 @@ export class DomainsWalletsGateway {
      },
    ])
   }
+
+  public addNewWallet(name_: string): Observable<IWalletApiResponse> {
+    return fakeRequest({
+      name: name_,
+      creationDate: '2022-10-22T09:47:52.595721658Z',
+      id: getRandomNumber(100, 10000),
+    });
+  }
+
+  public updateNewWallet(id_: number, name_: string): Observable<IWalletApiResponse> {
+    return fakeRequest({
+      name: name_,
+      creationDate: new Date().toString(),
```

Comment:
Please stick to one approach -> or change all `creationDate` to `new Date().toString()` or use a text like on line 34.